### DISCUSSION TITLE
Can this extension be installed using conda?

### DISCUSSION THREAD

**Discussion by stebsteb** [jorgensd/dolfinx_mpc#39] [Q&A] [answered]
Is the dolfinx_mpc extension available through conda?

**Comment by jorgensd** [comment]
> Is the dolfinx_mpc extension available through conda?

No. I tried to get this to work a while back, but got stuck with some obscure errors in conda recipe building. Maybe @minrk can help me fixing this, as he is the inhouse conda expert.

**Reply by minrk** [reply]
I can look at a recipe. Do you care to package Python and C++ separately?

**Reply by jorgensd** [reply]
It would be best to package C++ and Python separately, as I know there are some C++ only users of the code

**Reply by stebsteb** [reply]
Thanks. I don't think it matters to me whether they are packaged separately.

**Comment by jorgensd** [comment, marked as answer]
The extension is now on Conda: https://anaconda.org/conda-forge/dolfinx_mpc
